app/controllers/application_controller.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  include ErrorHandling
   include UserAuthorization
   include DeviseOverrides
   include TransformationReduxState
   include ExtractionReduxState
+  include LastEditedBy
```

Comment:
I just wonder if this is actually going to be used by EVERY controller, and if not, might be a bit clearer if you include the concern in each controller that needs it

Reply:
Done